### PR TITLE
Push arch specific images

### PR DESCRIPTION
**Pull request by code-elinka**
We have 3 registries for images:
* multiarch
* amd64
* arm64
 Even we can keep everything in the first one,
we still want to support last two registries
for users who may need that.

### CODE REVIEW

File: hack/push-multiarch.sh [hunk 1]
```diff
@@ -54,4 +54,15 @@ do
...
     echo "done, pushed $MULTIARCH_IMAGE image"
+
+    # Download crane cli
+    curl -sL "https://github.com/google/go-containerregistry/releases/download/v0.15.2/go-containerregistry_$(uname -s)_$(uname -m).tar.gz"
+    tar -zxvf go-containerregistry.tar.gz -C /usr/local/bin/ crane
+    
+    # Tag arch specific images for the legacy registries
+    for arch in ${ALL_ARCH}
+    do
+        crane copy --platform linux/${arch} ${MULTIARCH_IMAGE} ${REGISTRY}/ingress-gce-${binary}-${arch}:${VERSION}
```
Comment (aojea) [inline hack/push-multiarch.sh:65]: use `krane` https://github.com/google/go-containerregistry/blob/main/cmd/krane/README.md

File: hack/push-multiarch.sh [hunk 2]
```diff
@@ -54,4 +54,15 @@ do
         --tag  ${MULTIARCH_IMAGE} \
         -f Dockerfile.${binary} .
     echo "done, pushed $MULTIARCH_IMAGE image"
+
+    # Download crane cli
+    curl -sL "https://github.com/google/go-containerregistry/releases/download/v0.15.2/go-containerregistry_$(uname -s)_$(uname -m).tar.gz"
+    tar -zxvf go-containerregistry.tar.gz -C /usr/local/bin/ crane
```
Comment (aojea) [inline hack/push-multiarch.sh:60]:
```suggestion
curl -sL "https://github.com/google/go-containerregistry/releases/download/v0.15.2/go-containerregistry_$(uname -s)_$(uname -m).tar.gz" | tar xvzf - krane
```

File: hack/push-multiarch.sh (hunk 1 above)
Comment (aojea) [reply hack/push-multiarch.sh:65]:
```suggestion
        ./krane copy --platform linux/${arch} ${MULTIARCH_IMAGE} ${REGISTRY}/ingress-gce-${binary}-${arch}:${VERSION}
```

Comment (code-elinka) [reply hack/push-multiarch.sh]: done, thank you

File: hack/push-multiarch.sh (hunk 2 above)
Comment (code-elinka) [reply hack/push-multiarch.sh]: done, thank you

File: hack/push-multiarch.sh [hunk 3]
```diff
@@ -54,4 +54,15 @@ do
         --tag  ${MULTIARCH_IMAGE} \
         -f Dockerfile.${binary} .
     echo "done, pushed $MULTIARCH_IMAGE image"
+
+    # Download crane cli
+    curl -sL "https://github.com/google/go-containerregistry/releases/download/v0.15.2/go-containerregistry_$(uname -s)_$(uname -m).tar.gz" | tar xvzf - krane
```
Comment (aojea) [inline hack/push-multiarch.sh:59]: if you have multiple binaries this will download krane multiple times, you can put it at the beginning of the script

**Pull request merged**